### PR TITLE
Validation of new graph parameters

### LINKED ISSUE #267: Validation of new graph parameters
When inserting a new graph (issue #262), the window that collects the parameter must validate the value. This validation can be done with the help of networkx itself, according to the reference of each parameter (see #262 ).

When insert an invalid value:
- the "OK" must be disabled
- Below the parameter input text field will display red text indicating that the parameter is invalid.

### PR DESCRIPTION
**Pull request by fsoupimenta**
Closes #267

### CODE REVIEW

**Review by atilaajones — CHANGES_REQUESTED** [review]
- [ ] Adjust the "random cograph" screen design. The user enters the value k and a graph with 2^k vertices will be plotted, write this information.
- [x] The create button must be pre-selected by default, when you press enter it is activated;
- [x] Limit so that the number of vertices does not exceed 60, with natural check number;
- [ ] g6 needs to have a unique check, done just by clicking "create".
- [x] Replace link to definition graph6: http://users.cecs.anu.edu.au/~bdm/data/formats.txt
- [x] In star graph, replace _Number of nodes_ to _Number of leaves_
- [x] In Turan Graph, the software crashes if the user insert number of partitions before number of nodes. You can maybe to lock the second field, when the first field is empty.
- [x] In Grid 2D graph, replace the link to: https://en.wikipedia.org/wiki/Lattice_graph
- [x] In Triangular lattice graph, replace the link to: https://networkx.org/documentation/stable/reference/generated/networkx.generators.lattice.triangular_lattice_graph.html
- [x] In random regular graph, replace the link to: https://en.wikipedia.org/wiki/Regular_graph
- [ ] Insert a new graph option: Complete bipartite graph, see the third note [here](https://networkx.org/documentation/stable/reference/generated/networkx.generators.classic.complete_multipartite_graph.html)

File: source/store/new_graph_store.py [hunk 1]
```diff
@@ -13,227 +17,388 @@ def __init__(self):
...
+        dialog.graph_link.clicked.connect(lambda: NewGraphStore.open_url
+                                          ("https://en.wikipedia.org/wiki/Complete_bipartite_graph"))
+        dialog.dict['m'].textEdited.connect(lambda: NewGraphStore.verify_natural_number(dialog, 'm'))
+        dialog.dict['n'].textEdited.connect(lambda: NewGraphStore.verify_natural_number(dialog, 'n'))
+
+        NewGraphStore.open_new_dialog(dialog)
+
+    @staticmethod
+    def create_graph(dialog):
+        new_graph_store.set_graph(nx.complete_bipartite_graph(int(dialog.dict['m'].text()),
```
Comment (atilaajones) [inline source/store/new_graph_store.py:399]: Change to [Multipartite Complete](https://networkx.org/documentation/stable/reference/generated/networkx.generators.classic.complete_multipartite_graph.html#networkx.generators.classic.complete_multipartite_graph), with 2 sets.

**Review by atilaajones — CHANGES_REQUESTED** [review]

**Review by atilaajones — CHANGES_REQUESTED** [review]
In addition to the requested changes, make:

- [ ] Replace "<=" to \u2264
- [ ] Change the text (example) _"Number of leaves (n)"_, to _"Number of leaves, n"_. This change would be for all new_graph_store methods. What do you think?

File: source/store/new_graph_store.py [hunk 2]
```diff
@@ -13,227 +17,388 @@ def __init__(self):
...
         new_graph_store.set_graph(nx.Graph())
-        new_graph_store.set_file_path(project_information_store.project_location + f"\\{dialog.dict['name'].text()}.g6")
-
-        dialog.close()
+        NewGraphStore.create_graph(dialog)
 
 
 class GraphFromGraph6(NewGraphStore):
     name = "Graph from graph6"
+    dict_attributes_names = {"g6": "Graph from g6", "conditions": "graph6"}
```
[the remark this answers is not in the document]
Comment (fsoupimenta) [reply source/store/new_graph_store.py]: Done!

File: source/store/new_graph_store.py [hunk 3]
```diff
@@ -13,227 +17,388 @@ def __init__(self):
...
+            return
+        if validate_file_name(dialog.dict['name'].text()):
+            dialog.create_button.setEnabled(True)
+            dialog.dict['name'].setStyleSheet('background-color: white;')
+        else:
+            dialog.create_button.setDisabled(True)
+            dialog.dict['name'].setStyleSheet('background-color: #EF5350;')
+
+    @staticmethod
+    def verify_natural_number(dialog, param):
```
[the remark this answers is not in the document]
Comment (fsoupimenta) [reply source/store/new_graph_store.py]: Done!

File: source/store/new_graph_store.py (hunk 1 above)
Comment (fsoupimenta) [reply source/store/new_graph_store.py]: Done!

**Review by atilaajones — APPROVED** [review]

**Pull request merged**